package.json

```diff
@@ -29,7 +29,7 @@
     "LICENSE"
   ],
   "dependencies": {
-    "postcss": "^7.0.14"
+    "postcss": "^8.1.2"
```

Comment:
this should become a peer dependency, to respect the new rules of postcss

Reply:
I am not sure this is true for parsers (as opposed to plugins). You can see two parsers maintained by one of the main PostCSS maintainers put PostCSS in `dependencies`: https://github.com/postcss/postcss-safe-parser/blob/master/package.json and https://github.com/postcss/postcss-scss/blob/master/package.json
I think it’s because a parser imports PostCSS directly and essentially replaces PostCSS for the parsing step, so its dependency on PostCSS is a matter of internal implementation.

Reply:
the reason given in their blog post to make plugins use a peer dependency is to avoid issues related to mixing AST versions if the postcss dependency installs a different postcss version that the one used in the project. So to me, this also applies to parsers as they also build AST nodes.

Reply:
I'll comment on the main thread below, but moving it to `peerDeps` is not something I'm willing to do.